vinhloc30796/creativecontact-app
# style(header): shrink Creative Contact logo

## Description

Shrink Creative Contact logo on header and reorganize component files.

## Key Changes

1. Adjust Creative Contact logo size for better responsiveness
2. Move logo components to `components/branding/` directory
3. Relocate `Header.tsx` to `components/wrappers/` directory

## Testing Instructions

1. Open the event page on various screen sizes
2. Verify that the Creative Contact logo scales appropriately
3. Ensure that the header layout remains intact

## Additional Notes

This change improves the responsive design of the header while maintaining the overall layout structure.
